specification/cognitiveservices/data-plane/Language/preview/2022-07-01-preview/analyzetext.json

```diff
@@ -0,0 +1,2737 @@
+              "Operation-Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Job Delete Request": {
+            "$ref": "./examples/text/SuccessfulAnalyzeTextJobsCancelRequest.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "AnalyzeTextTaskKind": {
+      "type": "string",
+      "description": "Enumeration of supported Text Analysis tasks.",
+      "enum": [
+        "SentimentAnalysis",
+        "EntityRecognition",
+        "PiiEntityRecognition",
+        "KeyPhraseExtraction",
+        "LanguageDetection",
+        "EntityLinking"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeTextTaskKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeTextLROTaskKind": {
+      "type": "string",
+      "description": "Enumeration of supported long-running Text Analysis tasks.",
+      "enum": [
+        "SentimentAnalysis",
+        "EntityRecognition",
+        "PiiEntityRecognition",
+        "KeyPhraseExtraction",
+        "EntityLinking",
+        "Healthcare",
+        "ExtractiveSummarization",
+        "CustomEntityRecognition",
+        "CustomSingleLabelClassification",
+        "CustomMultiLabelClassification"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeTextLROTaskKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeTextTaskResultsKind": {
+      "type": "string",
+      "description": "Enumeration of supported Text Analysis task results.",
+      "enum": [
+        "SentimentAnalysisResults",
+        "EntityRecognitionResults",
+        "PiiEntityRecognitionResults",
+        "KeyPhraseExtractionResults",
+        "LanguageDetectionResults",
+        "EntityLinkingResults"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeTextTaskResultsKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeTextLROResultsKind": {
+      "type": "string",
+      "description": "Enumeration of supported Text Analysis long-running operation task results.",
+      "enum": [
+        "SentimentAnalysisLROResults",
+        "EntityRecognitionLROResults",
+        "PiiEntityRecognitionLROResults",
+        "KeyPhraseExtractionLROResults",
+        "EntityLinkingLROResults",
+        "HealthcareLROResults",
+        "ExtractiveSummarizationLROResults",
+        "CustomEntityRecognitionLROResults",
+        "CustomSingleLabelClassificationLROResults",
+        "CustomMultiLabelClassificationLROResults"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeTextLROResultsKind",
+        "modelAsString": true
+      }
+    },
+    "MultiLanguageAnalysisInput": {
+      "type": "object",
+      "properties": {
+        "documents": {
```

Comment:
should this be required?

Reply:
Same as below : Change was required by the linter - Linter fails otherwise.